plugins/linux-v4l2/linux-v4l2.c

```diff
@@ -15,18 +15,47 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 #include <obs-module.h>
+#include <util/platform.h>
 
 OBS_DECLARE_MODULE()
 OBS_MODULE_USE_DEFAULT_LOCALE("linux-v4l2", "en-US")
 MODULE_EXPORT const char *obs_module_description(void)
 {
-	return "Video4Linux2(V4L2) sources";
+	return "Video4Linux2(V4L2) sources/virtual camera";
 }
 
 extern struct obs_source_info v4l2_input;
+extern struct obs_output_info virtualcam_info;
+
+static bool v4l2loopback_installed()
+{
+	bool loaded = false;
+
+	int ret = system("modinfo v4l2loopback");
```

Comment:
Why is there not used the "pkexec" call for polkit like line 56?
On Debian modinfo is not usr callable.

Reply:
modinfo does not require root permissions, but in some distributions it is installed in `/sbin`, so OBS can not rely on the binary being in `PATH`

Reply:
Discussions on locating binaries like this, and possibly using different paths, can also be found in https://github.com/obsproject/obs-studio/pull/3182#discussion_r496062890. Mentioning this so we don't keep repeating the exact same discussions.